coverage_comment/default.md.j2

```diff
@@ -28,7 +33,11 @@ The branch rate is `{{ (coverage.info.covered_branches / coverage.info.num_branc
 {% endblock branch_coverage -%}
 
 {% block diff_coverage_wording -%}
+{%- if diff_coverage.total_num_lines > 0 -%}
 `{{ diff_coverage.total_percent_covered | pct }}` of new lines are covered.
+{%- else -%}
+_None of the new lines are part of the tested code. Therefore, there is no coverage data about them._
+{%- endif %}
```

Comment:
The `{% block diff_coverage_wording -%}` should be inside the block, I think, not outside. And we probably need to introduce another block, like `{% block diff_coverage_empty_wording -%}`